src/java.base/share/classes/java/nio/channels/FileChannel.java

```diff
@@ -997,6 +997,10 @@ public abstract MappedByteBuffer map(MapMode mode, long position, long size)
      *           of mapped memory associated with the returned mapped memory
      *           segment is unspecified and should not be relied upon.
      *
+     * @implSpec The {@code map(MapMode, long, long, MemorySession)} method of
```

Comment:
This looks a bit strange to me, because for the reader it looks like the default file channels in Java also does this. I'd add some other sentence part explaining this.

Alternative: "The default implementation of `map(MapMode, long, long, MemorySession)` method in abstract `FileChannel` always throws an `UnsupportedOperationException`. Subclasses should override it."